org-code-javabuilder/validation/src/main/java/org/code/validation/support/ValidationRuntimeException.java

```diff
@@ -0,0 +1,35 @@
+package org.code.validation.support;
+
+import org.code.protocol.JavabuilderRuntimeException;
+
+public class ValidationRuntimeException extends JavabuilderRuntimeException {
+  protected ValidationRuntimeException(ExceptionKey key) {
+    super(key);
+    this.setFallbackMessage(this.generateFallbackMessage(key, null));
+  }
+
+  protected ValidationRuntimeException(ExceptionKey key, Throwable cause) {
+    super(key, cause);
+    this.setFallbackMessage(this.generateFallbackMessage(key, cause));
```

Comment:
Could we just add a constructor to `JavabuilderRuntimeException` that takes `key`, `cause`, and `fallbackMessage` so we could do `super(key, cause, this.generateFallbackMessage(key, cause))`? That way we don't have to expose `setFallbackMessage` on JavabuilderRuntimeException?

Reply:
unfortunately we can't because you have to call super() before anything else. Since the method call for the parameter gets called before super() actually gets called you get a compilation error.

Reply:
Ahh right, I wasn't sure if calling the method inline would work, but yeah makes sense.